tools/pkglint/lib/rules.ts

```diff
@@ -376,11 +376,11 @@ export class NodeCompatibility extends ValidationRule {
 
   public validate(pkg: PackageJson): void {
     const atTypesNode = pkg.getDevDependency('@types/node');
-    if (atTypesNode && !atTypesNode.startsWith('^12.')) {
+    if (atTypesNode && !atTypesNode.startsWith('^14.')) {
       pkg.report({
         ruleName: this.name,
-        message: `packages must support node version 12 and up, but ${atTypesNode} is declared`,
-        fix: () => pkg.addDevDependency('@types/node', '^12.18.3'),
+        message: `packages must support node version 14 and up, but ${atTypesNode} is declared`,
+        fix: () => pkg.addDevDependency('@types/node', '^14.14.31'),
```

Comment:
should this be `14.15.0`?

Reply:
We discussed offline and this is okay. There are no newer type stubs for 14 and Node 14.15.0 didn't have any significant changes over 14.14.x ([ref](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V14.md#14.15.0)).